src/Database/Queries/Roles.php

```diff
@@ -40,6 +56,22 @@ public function constrainWhereIsAll($query, $role)
     }
 
     /**
+     * Constrain the given query by all provided roles.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  string  $role
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function constrainWhereIsNotAll($query, $role)
+    {
+        $roles = array_slice(func_get_args(), 1);
+
+        return $query->whereHas('roles', function ($query) use ($roles) {
+            $query->whereInNot('name', $roles);
```

Comment:
hi Joseph
I appreciate you very much for this amazing package 👍 
And thanks for your patience in my mistake.
(sorry for my bad english)